calblueprint/forte
# [Mailer] E-mail teacher/student + admin upon completing signup form

Fix #172 
Fix #189 

![image](https://cloud.githubusercontent.com/assets/5432430/20516392/3623a31a-b04b-11e6-8a1c-547ac0b0c948.png)
![image](https://cloud.githubusercontent.com/assets/5432430/20516394/3b247024-b04b-11e6-9519-99ee1db028b1.png)
![image](https://cloud.githubusercontent.com/assets/5432430/20516401/3f8a95e4-b04b-11e6-84f8-19d9af63d800.png)


Testing Signup Procedures:
1. Fill out sign up form & submit 
2. Email should pop up 

Testing Contact Procedures:
1. Fill out contact form & submit 
2. Email should pop up
